e2e/pageobjects/ide/Editor.ts

```diff
@@ -296,6 +298,39 @@ export class Editor {
         return lineYCoordinate;
     }
 
+    async clickOnTheTextInDefinedPosition(line: number, column: number) {
```

Comment:
Method name could be simplified to `clickOnTheSymbol`.

Reply:
`moveCursorToLineAndChar()` implements setting up cursor with keyboard, but  `clickOnTheTextInDefinedPosition` move mouse cursor by x and y coordinates. It is because this one does not need selecting of the editor tab file.